web/controllers/courseController/index.ts

```diff
@@ -2772,7 +2772,8 @@ export async function groupMailHandler(req: Request, res: Response) {
     res.status(status).end();
 }
 
-async function groupMail(student: Student, courseId: number, subcourseId: number, mailSubject: string, mailBody: string) {
+async function groupMail(student: Student, courseId: number, subcourseId: number, mailSubject: string, mailBody: string, rawAddressees: string[]) {
+    console.log("DEBUG", "Addressees", rawAddressees);
```

Comment:
Please use `getLogger("Courses API").debug("groupMail adresses")`, a "DEBUG" in the logs is hard to track back to the origin ...